2022-10-10 22:40:13,595 INFO    MainThread:3355427 [wandb_setup.py:_flush():68] Configure stats pid to 3355427
2022-10-10 22:40:13,595 INFO    MainThread:3355427 [wandb_setup.py:_flush():68] Loading settings from /home/ashwinee/.config/wandb/settings
2022-10-10 22:40:13,595 INFO    MainThread:3355427 [wandb_setup.py:_flush():68] Loading settings from /scratch/gpfs/ashwinee/dp_finetuning/wandb/settings
2022-10-10 22:40:13,595 INFO    MainThread:3355427 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-10-10 22:40:13,595 INFO    MainThread:3355427 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'finetune_classifier_dp.py', 'program': 'finetune_classifier_dp.py'}
2022-10-10 22:40:13,595 INFO    MainThread:3355427 [wandb_init.py:_log_setup():476] Logging user logs to /scratch/gpfs/ashwinee/dp_finetuning/wandb/run-20221010_224013-qmdzwbw8/logs/debug.log
2022-10-10 22:40:13,595 INFO    MainThread:3355427 [wandb_init.py:_log_setup():477] Logging internal logs to /scratch/gpfs/ashwinee/dp_finetuning/wandb/run-20221010_224013-qmdzwbw8/logs/debug-internal.log
2022-10-10 22:40:13,595 INFO    MainThread:3355427 [wandb_init.py:init():516] calling init triggers
2022-10-10 22:40:13,595 INFO    MainThread:3355427 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-10-10 22:40:13,596 INFO    MainThread:3355427 [wandb_init.py:init():569] starting backend
2022-10-10 22:40:13,596 INFO    MainThread:3355427 [wandb_init.py:init():573] setting up manager
2022-10-10 22:40:13,599 INFO    MainThread:3355427 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-10-10 22:40:13,603 INFO    MainThread:3355427 [wandb_init.py:init():580] backend started and connected
2022-10-10 22:40:13,607 INFO    MainThread:3355427 [wandb_init.py:init():658] updated telemetry
2022-10-10 22:40:13,607 INFO    MainThread:3355427 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-10-10 22:41:13,621 ERROR   MainThread:3355427 [wandb_init.py:init():704] backend process timed out
2022-10-10 22:41:13,621 ERROR   MainThread:3355427 [wandb_init.py:init():713] encountered error: Error communicating with wandb process
For more info see: https://docs.wandb.ai/library/init#init-start-error
